docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -143,6 +143,9 @@ Key interfaces
 
     .. versionadded:: 0.9
 
+    DH private key objects have methods from this interface as well as
+    :class:`DHPrivateKeyWithSerialization`.
```

Comment:
I can't come up with a clear way to say "Technically private keys may not implement serialization in the future but for now they all do and even when that's not true your code won't care unless you add support for opaque keys to your application".

How about `DH private key objects typically also implement DHPrivateKeyWithSerialization to provide serialization methods`?

The one problem with that phrasing is that we know words like "typically" or "usually" results in user confusion (see: 65537 on RSA generation).